prime-dotnet-webapi/Services/EmailRenderingService.cs

```diff
@@ -161,7 +161,7 @@ public async Task<Email> RenderSiteRegistrationSubmissionEmailAsync(LinkedEmailV
                 from: PrimeEmail,
                 to: MohEmail,
                 cc: PrimeSupportEmail,
-                subject: "PRIME Site Registration Submission",
+                subject: $"[{siteId}] PRIME Site Registration Submission",
```

Comment:
Just want to confirm that adding the `siteId` is **not** applicable to Community Pharmacy sites?

Reply:
> Just want to confirm that adding the `siteId` is **not** applicable to Community Pharmacy sites?

I believe the requirement was for emails sent to hibc, whereas Community Pharmacy sites emails are sent to PrimeSupport.